packages/jellyfish-transaction/src/buffer/buffer_composer.ts

```diff
@@ -438,4 +439,25 @@ export abstract class ComposableBuffer<T> implements BufferComposer {
       }
     }
   }
+
+  static mask (
+    getter: () => boolean[],
+    setter: (data: boolean[]) => void
+  ): BufferComposer {
+    return {
+      fromBuffer: (buffer: SmartBuffer): void => {
+        const num = buffer.readInt8()
+        const array: boolean[] = []
+        for (let i = 0; i < getter().length; i += 1) {
```

Comment:
For `from` operation you should use `getter()`, if you need the length you should add it to the required parameter.

```ts
    length: number,
    getter: () => boolean[],
    setter: (data: boolean[]) => void
```